lib/merkle/generateMerkleProof.ts

```diff
@@ -0,0 +1,30 @@
+import MerkleTree from "merkletreejs"
```

Comment:
we can prob move this to api, make it easier to get root and proof. I am assuming you would have to do this every time someone mints on cre8orlist day. 
make an api that takes a param of address and move all of this generation to server side.

Reply:
migrated merkle proof generation to `/api/merkle?address={X}` ✅